### PR TITLE
Completed Prep Javascript Conditionals Exercise - Allyson Greven

### PR DESCRIPTION
**Pull request by allygreven**
Screenshot 2024-10-05 at 2 29 32 PM

### CODE REVIEW

File: prep-javascript-conditionals/main.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+/* Your JS goes in this file */
+function isAdult(age) {
+  if (age >= 18) {
+    return true;
+  }
+  if (age <= 18) {
```
Comment (robgmerrill) [inline prep-javascript-conditionals/main.js]: Close! 18 would be an adult. So in this case you would want to check if `age < 18` to confirm that they are not an adult.

File: prep-javascript-conditionals/main.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  }
+}
+const isAdultResult = isAdult(23);
+console.log(isAdultResult);
+///////////////////////////////////////////////////
+function didStudentPass(score) {
+  if (score >= 70) {
+    return true;
+  }
+  if (score <= 70) {
```
Comment (robgmerrill) [inline prep-javascript-conditionals/main.js]: Same as above comment :)

File: prep-javascript-conditionals/main.js (hunk 1 above)
Comment (robgmerrill) [reply prep-javascript-conditionals/main.js]: or `age <= 17`

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Couple small changes requested :)

**Comment by allygreven** [comment]
Screenshot 2024-10-08 at 5 38 53 PM

**Comment by allygreven** [comment]
after going over the exercise, I know where I messed up... again. Will fix and resubmit 👍🏼

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**